chaos/actor.go

```diff
@@ -37,6 +37,18 @@ var (
 	CallerValidationBranchTypeNilSet = big.NewIntUnsigned(3)
 )
 
+// MutateStateType is an enum used to select the type of state mutation to attempt.
+type MutateStateType uint64
```

Comment:
Mind calling this `MutateStateBranch`, for consistency with other similar methods?